Formula/p/poac.rb

```diff
@@ -41,23 +31,13 @@ class Poac < Formula
 
   def install
     ENV.llvm_clang if OS.mac? && (DevelopmentTools.clang_build_version <= 1200)
-
-    # Help to find OpenSSL.
-    inreplace "cmake/AddOpenSSL.cmake", "${POAC_HOMEBREW_ROOT_PATH}/openssl",
-                                        Formula["openssl@3"].opt_prefix
-
-    system "cmake", "-S", ".", "-B", "build", "-DPOAC_BUILD_TESTING=OFF", *std_cmake_args
-    system "cmake", "--build", "build"
-    system "cmake", "--install", "build"
-
-    man.install "src/etc/man/man1"
-    bash_completion.install "src/etc/poac.bash" => "poac"
-    zsh_completion.install_symlink bash_completion/"poac" => "_poac"
```

Comment:
Are these supported in the new build?

Reply:
@chenrui333 Unfortunately, these are removed to reduce maintenance costs. That said, I am willing to support these in future releases.

Reply:
got you, thanks!